app/controllers/users_controller.rb

```diff
@@ -0,0 +1,16 @@
+class UsersController < ApplicationController
+  def register
+    @user = User.new
+    render :register
+  end
+
+  def create
+    @user = User.create(params.require(:user).permit(:full_name, :email, :password))
```

Comment:
Extracting string params to a method contains the settings nicely and cleans up your code a bit.